ufmt/tests/core.py

```diff
@@ -18,6 +19,11 @@
     "foo/frob/",
     "__init__.py",
 ]
+
+[tool.black]
+target_version = ["py36", "py37"]
+skip_string_normalization = true
```

Comment:
This is problematic. I've included it here, since this is not the default behavior of `black` and thus we can be sure when we see it in the config that the file has been read. On the other hand all the format tests now also lead to the failures, because strings are no longer normalized, i.e. `'foo'` -> `"foo"`.  

It doesn't seem to be the case, but changing the default line length could have the same effect. How do you want me to fix it? I could either remove this line or add the `black` config dynamically.

Reply:
If there's not a single fake config that works, maybe it would be easier to have multiple fake config values? Otherwise, I'd prefer if you could use [tomlkit](https://pypi.org/project/tomlkit/) to generate the config from a dictionary; that's already a dependency, and should reduce the amount of code necessary to turn a dictionary or table object into a string that can be written to a file.

Reply:
Right now there is only a single test that uses a `black` config and it can simply append to the regular `FAKE_CONFIG`. Thus, I've opted for your second suggestion. Let me know if you want to have a static config after all.